src/ensembl/src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts

```diff
@@ -69,9 +70,46 @@ const defaultStatePerGene: TranscriptsStatePerGene = {
   expandedDownloadIds: [],
   expandedMoreInfoIds: [],
   filters: {},
-  sortingRule: SortingRule.DEFAULT
+  sortingRule: SortingRule.DEFAULT,
+  filterPanelOpen: false
 };
 
+export const resetFilterPanel =
+  (): ThunkAction<void, any, null, Action<string>> =>
+  (dispatch, getState: () => RootState) => {
+    const state = getState();
+    const activeGenomeId = getEntityViewerActiveGenomeId(state);
+    const activeEntityId = getEntityViewerActiveEntityId(state);
+    if (!activeGenomeId || !activeEntityId) {
+      return;
+    }
+    dispatch(
+      transcriptsSlice.actions.updateFilterPanel({
+        activeGenomeId,
+        activeEntityId,
+        filterPanelOpen: defaultStatePerGene.filterPanelOpen
+      })
+    );
+  };
+
+export const setFilterPanel =
```

Comment:
`setFilterPanel` -> `setFilterOpenState?`?

Reply:
I like the word "panel" in "filter panel" 😞 It is distinct from the actual filters, which we are updating [here](https://github.com/Ensembl/ensembl-client/blob/77725aacdbd9998bc76f4d203105f1506ceadee2/src/ensembl/src/content/app/entity-viewer/state/gene-view/transcripts/geneViewTranscriptsSlice.ts#L325-L333)

Reply:
I agree that it is distinct but using `setFilterPanel` sounds like we are doing more than just setting the `open` state.